openquake/calculators/tests/classical_test.py

```diff
@@ -566,3 +566,33 @@ def test_case_47(self):
         # Mixture Model for Sigma using PEER (2018) Test Case 2.5b
         self.assert_curves_ok(["hazard_curve-rlz-000-PGA.csv"],
                               case_47.__file__)
+
+    def test_case_48(self):
+        # pointsource_distance effects on a simple point source
+        self.run_calc(case_48.__file__, 'job.ini')
+        tmp = general.gettemp(rst_table(self.calc.datastore['rup/rrup_'],
+                                        ['sid0', 'sid1']))
+        self.assertEqualFiles('expected/exact_dists.txt', tmp)
+
+        self.run_calc(case_48.__file__, 'job.ini', pointsource_distance='*')
+        tmp = general.gettemp(rst_table(self.calc.datastore['rup/rrup_'],
+                                        ['sid0', 'sid1']))
+        self.assertEqualFiles('expected/approx_dists.txt', tmp)
+        # this test shows in detail what happens to the distances in presence
+        # of a magnitude-dependent pointsource_distance: just look at the
+        # files expected/exact_dists.txt and expected/approx_dists.txt
+        # the exact distances for the first site are 54, 53, 53, ... 38, 32 km
+        # they decrease with the magnitude, since big magnitude -> big size ->
+        # smaller distance from the site.
+        # When the pointsource_distance is on, the approximated distances are
+        # 55, 55, 55, ..., 38, 32 km: the difference is in the first three
+        # values, corresponding to the small magnitudes.
+        # For small magnitudes the planar ruptures are replaced by points
+        # and thus the distances become larger (and constant).
+        # The maximum_distance here is 110 km and the second site
+        # was chosen very carefully, so that the exact distance for the highest
+        # magnitude is 109 km (within) while the approx distance is 111 km
+        # (outside): still the rupture is not discarded because we are in the
+        # few-sites regime.
```

Comment:
You need one phrase explaining the differences between the two regimes.

Reply:
The two regimes are documented here: https://github.com/gem/oq-engine/blob/master/doc/adv-manual/special-features.rst#max_sites_disagg